subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -328,8 +329,27 @@ static void do_bootstrap_reg_timeout_cb(struct lwm2m_message *msg)
 	/* Restart from scratch */
 	sm_handle_timeout_state(msg, ENGINE_INIT);
 }
+
 #endif
 
+int engine_trigger_bootstrap(void)
+{
+#if defined(CONFIG_LWM2M_RD_CLIENT_SUPPORT_BOOTSTRAP)
+	if (!sm_is_registered()) {
+		/* Bootstrap is not possible to trig */
+		LOG_WRN("Not Possible to Trig Bootstrap from state %u ", client.engine_state);
+		return -EPERM;
+	}
+
+	LOG_INF("Bootstrap initiated by server /1x/9 Exececute");
```

Comment:
I think the resource path can be left out, and there is a typo anyway.
So preferably change this to `Server Initiated Bootstrap`

Reply:
OK I will change.